patroni/dcs/__init__.py

```diff
@@ -535,6 +535,10 @@ def timeline(self):
                 return 1
         return 0
 
+    @property
+    def max_timelines_history(self):
+        return self.config.data.get('max_timelines_history', 0)
+
```

Comment:
The `max_timelines_history` is a configuration parameter, so it is more appropriate to put it into the [ClusterConfig](https://github.com/zalando/patroni/pull/1491/files#diff-0964cdd11e7e37271af75805b15fbcd2R319) class.

Reply:
Thanks. Updating the code.